src/index.html

```diff
@@ -2,11 +2,54 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        initial-scale=1.0"
+    >
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div
+      class="
+        calendar
+        page__calendar
+        calendar--start-day--sun
+        calendar--month-length--29"
+    >
```

Comment:
this doesn't open in the demo link, i don't know why but there is also github failure error for this.

Reply:
The link works fine in my end, the github failure I think is because the no test on this task.